### DISCUSSION TITLE
maintainers ig

### DISCUSSION THREAD

**Discussion by kheina** [kheina-com/Blue-Blocker#121] [General]
I don't know where else to put this so I guess the discussions tab exists now, but I added yall as writers to the repo. hopefully this will prevent stuff from standing still completely while I'm dealing with burnout and shit @jaredkotoff @mrawesome. I think you guys can merge to main now with a review which I think you guys can also provide but I'll try to stay on top of reviews. would be cool to make this not a solo operation anymore

**Comment by jaredcat** [comment]
Awesome, thank you. 
Yea, forcus on your mental health. The extension does its job, so I don't think any improvements need to be rushed anyways.
